doc/releases/changelog-0.23.0.md

```diff
@@ -1,14 +1,179 @@
 :orphan:
 
-# Release 0.23.0-dev (development release)
+# Release 0.23.0-dev (current release)
 
 <h3>New features since last release</h3>
 
-* Added an optimization transform that matches pieces of user-provided identity templates in a circuit and replaces them with an equivalent component.
+<h4> More powerful circuit cutting ✂️</h4>
+
+* Quantum circuit cutting (running `N`-wire circuits on devices with fewer than
+  `N` wires) is now supported for QNodes of finite-shots using the new
+  `@qml.cut_circuit_mc` transform.
+  [(#2313)](https://github.com/PennyLaneAI/pennylane/pull/2313)
+  [(#2321)](https://github.com/PennyLaneAI/pennylane/pull/2321)
+  [(#2332)](https://github.com/PennyLaneAI/pennylane/pull/2332)
+  [(#2358)](https://github.com/PennyLaneAI/pennylane/pull/2358)
+  [(#2382)](https://github.com/PennyLaneAI/pennylane/pull/2382)
+  [(#2399)](https://github.com/PennyLaneAI/pennylane/pull/2399)
+  [(#2407)](https://github.com/PennyLaneAI/pennylane/pull/2407)
+  [(#2444)](https://github.com/PennyLaneAI/pennylane/pull/2444)
+
+  With these new additions, samples from the original circuit can be
+  simulated using a Monte Carlo method, using fewer qubits at the expense of
+  more device executions. Additionally, this transform can take an optional
+  classical processing function as an argument and return an expectation
+  value.
+
+  The following `3`-qubit circuit contains a `WireCut` operation and a `sample`
+  measurement. When decorated with `@qml.cut_circuit_mc`, we can cut the circuit
+  into two `2`-qubit fragments:
+
+  ```python
+  dev = qml.device("default.qubit", wires=2, shots=1000)
+
+  @qml.cut_circuit_mc
+  @qml.qnode(dev)
+  def circuit(x):
+      qml.RX(0.89, wires=0)
+      qml.RY(0.5, wires=1)
+      qml.RX(1.3, wires=2)
+
+      qml.CNOT(wires=[0, 1])
+      qml.WireCut(wires=1)
+      qml.CNOT(wires=[1, 2])
+
+      qml.RX(x, wires=0)
+      qml.RY(0.7, wires=1)
+      qml.RX(2.3, wires=2)
+      return qml.sample(wires=[0, 2])
+  ```
+
+  we can then execute the circuit as usual by calling the QNode:
+
+  ```pycon
+  >>> x = 0.3
+  >>> circuit(x)
+  tensor([[1, 1],
+          [0, 1],
+          [0, 1],
+          ...,
+          [0, 1],
+          [0, 1],
+          [0, 1]], requires_grad=True)
+  ```
+
+  Furthermore, the number of shots can be temporarily altered when calling
+  the QNode:
+
+  ```pycon
+  >>> results = circuit(x, shots=123)
+  >>> results.shape
+  (123, 2)
+  ```
+
+  The `cut_circuit_mc` transform also supports returning sample-based
+  expectation values of observables using the `classical_processing_fn`
+  argument. Refer to the `UsageDetails` section of the [`cut_circuit_mc`
+  transform
+  documentation](https://pennylane.readthedocs.io/en/latest/code/api/pennylane.cut_circuit_mc.html)
```

Comment:
```suggestion
  argument. Refer to the Usage Details section of the [documentation](https://pennylane.readthedocs.io/en/latest/code/api/pennylane.cut_circuit_mc.html)
```

(since ReST doesn't support nested formatting :( )

![image](https://user-images.githubusercontent.com/2959003/165165977-ef6d61e6-5fe8-4f6c-a2fe-f58836590a63.png)